dist2src/cli.py

```diff
@@ -176,6 +176,29 @@ def get_archive(gitdir):
     return stdout
 
 
+class D2SSpecFile(SpecFile):
+    """ add a way to comment patches out in a spec file """
+
+    def comment_patches(self):
+        applied_patches = self.get_applied_patches()
+        patch_indices = [p.index for p in applied_patches]
+
+        pattern = re.compile(r"^Patch(?P<index>\d+)\s*:.+$")
+        package = self.spec_content.section("%package")
+        for i, line in enumerate(package):
+            match = pattern.match(line)
+            if match:
```

Comment:
I was thinking about creating a new commit for each match, but the changes from patches are already committed in the source-git in this phase, aren't they?

Reply:
> I was thinking about creating a new commit for each match

I'm sorry but I don't follow.

> patches are already committed in the source-git

correct

Reply:
> > I was thinking about creating a new commit for each match
> 
> I'm sorry but I don't follow.

No problem, I already answered to myself that it's not possible..;)

(I wanted to do the patch commits here and then realised that we have already converted those changes elsewhere...)